core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -44,13 +44,16 @@ angular.module('oppia').directive('ckEditor4Rte', [
           ContextService.getPageContext() === PAGE_CONTEXT.TOPIC_EDITOR ||
           ContextService.getPageContext() === PAGE_CONTEXT.STORY_EDITOR ||
           ContextService.getPageContext() === PAGE_CONTEXT.SKILL_EDITOR);
+        var contextIsExplorationPlayer = (
+          ContextService.getPageContext() === PAGE_CONTEXT.EXPLORATION_PLAYER);
 
         _RICH_TEXT_COMPONENTS.forEach(function(componentDefn) {
           if (!((scope.uiConfig() &&
             scope.uiConfig().hide_complex_extensions &&
             componentDefn.isComplex) ||
             (!canUseFs && componentDefn.requiresFs) ||
-            (!contextIsLessonRelated && componentDefn.isLessonRelated))) {
+            (!contextIsLessonRelated && componentDefn.isLessonRelated)) ||
+            (contextIsExplorationPlayer && componentDefn.id === 'image')) {
```

Comment:
If we're treating all the components the same now, there should not be any special-casing of 'image' in this PR, right?

Reply:
changed it